versioned_sidebars/version-5-sidebars.json

```
@@ -23,6 +23,7 @@
           "label": "Install XState",
           "id": "installation"
         },
+        "quickstart",
```

I usually see these words separated in docs (nit):

```suggestion
        "quick-start",
```